### DISCUSSION TITLE
Unified room list instead of separating DMs and groups

### DISCUSSION THREAD

**Discussion by wuast94** [element-hq/element-meta#2667] [Ideas]
An option to create an unified room list would be awesome. splitting rooms and direct messages isnt really usable, and option to make one view where anything is in it would be nice. like almost every other messanger does.

**Comment by awtj8o81ryywg793** [comment]
With Element X, unless you filter the room list, it will display all rooms (including direct messages) sorted in order of most recently updated.

**Comment by cipherswami** [comment]
oh man not really sure what you are talking. I was actually trying proposing the quite opposite of what you said. The rooms and DM's are already in the All chats section.

FYI: Using element classic and web.

**Reply by awtj8o81ryywg793** [reply]
In regards to Element Web, the room list design isn't finalised, and will likely be revisited.

**Comment by cipherswami** [comment]
I was thinking like lets keep it simple. 

All DM's (1:1 Room) and Groups (General Rooms) in the all chats section. But rooms from spaces are not be in this all chats section as it being cluttered on the main view.

This rooms from spaces usually tend to be noisy and are not important groups as well. This is already being done in the web/desktop. Just need to be done on mobile version as well.

**Reply by awtj8o81ryywg793** [reply]
Quoted from [parent comment](https://github.com/element-hq/element-meta/discussions/2667#discussioncomment-14724001):

> rooms from spaces usually tend to be noisy and are not important groups as well.

What do you mean by "noisy"? If you’re referring to notifications, you can configure per-room notification settings to make things a bit quieter.

Quoted from [parent comment](https://github.com/element-hq/element-meta/discussions/2667#discussioncomment-14724001):

> This is already being done in the web/desktop. Just need to be done on mobile version as well.

You might want to watch the "[Element X and Pro Updates](https://www.youtube.com/watch?v=_cahXxr8d-4)" talk from the 2025 Matrix Conference. Skip to [23:15](https://www.youtube.com/watch?v=_cahXxr8d-4&t=1395) in the talk to see a demonstration of what’s to come regarding spaces in Element X. Based on that, I would appreciate your thoughts on this.

**Reply by cipherswami** [reply]
Damn that is nice, it would be far better if all those spaces are moved into the spaces category down there.

I'm on the standard end user version. So, I don't see it yet.

**Reply by cipherswami** [reply]
> What do you mean by "noisy"? If you’re referring to notifications, you can configure per-room notification settings to make things a bit quieter.

Noisy in the sense these rooms are active but they are not that important. Just like discord channels. Let's say if I joined more spaces and rooms in it, these rooms occupy all of my chats section making it difficult for me to see the real import chats like DM's.

**Reply by awtj8o81ryywg793** [reply]
Quoted from a [previous reply](https://github.com/element-hq/element-meta/discussions/2667#discussioncomment-14732009):

> it would be far better if all those spaces are moved into the spaces category down there.

In which way would it be better? Personally, I find the current behaviour of Element Web/Desktop rather unintuitive. If you've joined a room, why wouldn't it be included in the room list? I'm in favour of bringing the ideas showcased in the previously linked talk to Element Web/Desktop.

**Reply by awtj8o81ryywg793** [reply]
Quoted from a [previous reply](https://github.com/element-hq/element-meta/discussions/2667#discussioncomment-14732135):

> these rooms occupy all of my chats section making it difficult for me to see the real import chats like DM's.

Couldn't you simply mark those rooms as favourites or use the filters? Personally, I wouldn’t rely on an unfiltered room list to find important rooms, as it’s just a list of all the rooms you’re in, organised by the most recently updated.

**Comment by cipherswami** [comment]
I was thinking like this:

All chats Section -> DMs, Rooms (rooms not part of space)
Spaces Section -> Joined Spaces -> Joined rooms [Three dots to give more options of space like listing all the available rooms and space settings etc.]

And in general settings they should give an option to include rooms of spaces in all chats.
